benchmarks/docker/django_simple/script.js

```diff
@@ -0,0 +1,6 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+    const res = http.get('http://127.0.0.1:5000/accounts/signup/');
```

Comment:
what does this endpoint exercise compared to others?

Reply:
Some db calls but mostly templating. I had about 70 spans whereas the main page was half that. Being that we are focusing here on putting the tracer under load, I went with this route.

![image](https://user-images.githubusercontent.com/155673/122466422-49c34280-cf87-11eb-9ebc-73f3e4ff02f8.png)

Reply:
cool, good choice!